index.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+require_once 'database.php';
```

Comment:
dont need a database.php file, move the functions to functions.php and call the functions at the top of index.php